.github/dependabot.yml

```
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: "gradle"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+  - package-ecosystem: "github_actions"
```

```suggestion
  - package-ecosystem: "github-actions"
```
Could you please change it to a dash? From the documentation: https://docs.github.com/en/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot

---

Thank you for the catch. I confused them because that's the name internally. Thanks for the review!